cmd/dynamic_startup.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/state/protocol/inmem"
+)
+
+const sleepInterval = time.Minute
```

Comment:
We could make this a flag as well, though I think most people would be fine with a 1-minute default.

Reply:
https://github.com/onflow/flow-go/pull/1893/commits/8ea656ac507e9969c635a600e908891394d7ac2c